docs/fairness-indicators.md

```diff
@@ -0,0 +1,58 @@
+# Evaluating Models with the Fairness Indicators Dashboard [Beta]
+
+![Fairness Indicators](./images/fairness-indicators.png)
+
+Fairness Indicators for TensorBoard enables easy computation of commonly-identified fairness metrics for _binary_ and _multiclass_ classifiers. With the plugin, you can visualize fairness evaluations for your runs and easily compare performance across groups. 
+
+In particular, Fairness Indicators for TensorBoard includes the ability to:
+* Evaluate the distribution of datasets
+* Evaluate model performance, sliced across defined groups of users. Feel confident about your results with confidence intervals and evaluations at multiple thresholds 
+
+Many existing tools for evaluating fairness concerns don’t work well on large scale datasets and models. At Google, it is important for us to have tools that can work on billion-user systems. Fairness Indicators will allow you to evaluate across any size of use case, in the Tensorboard environment or in [Colab](https://github.com/tensorflow/fairness-indicators). 
+
+## Requirements
+
+To install Fairness Indicators for Tensorboard, run:
+
+```
+python3 -m virtualenv ~/tensorboard_demo
+source ~/tensorboard_demo/bin/activate
+pip install tensorboard_plugin_fairness_indicators
+pip uninstall -y tensorflow tf-nightly
+pip install "tensorflow_model_analysis>=0.15.1"
+pip uninstall -y tensorboard tb-nightly
+pip install "tb-nightly>=2.1.0a20191024"
+```
+
+## Demo
+
+If you want to test out Fairness Indicators in TensorBoard, you can download sample TensorFlow Model Analysis evaluation results (eval_config, metrics and plots files) and a `demo.py` utility from Google Cloud Platform, [here](https://console.cloud.google.com/storage/browser/tensorboard_plugin_fairness_indicators/). This evaluation data is based on the [Civil Comments dataset](https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification), calculated using Tensorflow Model Analysis's [model_eval_lib](https://github.com/tensorflow/model-analysis/blob/master/tensorflow_model_analysis/api/model_eval_lib.py) library. It also contains a sample TensorBoard summary data file for reference. See the [TensorBoard tutorial](https://github.com/tensorflow/tensorboard/blob/master/README.md) for more information on summary data files.
```

Comment:
Speaking of obvious but to be thorough since you have linked it twice (but to different urls), you can write something like this:

``````markdown
....blah blah [Google Cloud Platform] blahblah...
....another passage [Google Cloud Platform] blahblah...

...somewhere in the document (preferably at the end)...
[Google Cloud Platform]: https://console.cloud.google.com/storage/browser/tensorboard_plugin_fairness_indicators/
``````

Both instances of `[Google Cloud Platform]` will link to the same url.

Reply:
Thanks for pointing out the two different links. Updated them. And I guess attaching link inline might be better in terms of usage else they have to look for the link in the document.